### PR TITLE
.NET client for Intune Service

### PR DESCRIPTION
**Pull request by vrmurthy01**
First version of .NET client code with unit tests for Intune service

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@vrmurthy01__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (vrmurthy). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by amarzavery** [comment]
@azuresdkci Add to whitelist

File: src/ResourceManagement/Intune/.vs/config/applicationhost.config [hunk 1]
```diff
@@ -0,0 +1,1030 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<!--
```
Comment (amarzavery) [inline src/ResourceManagement/Intune/.vs/config/applicationhost.config]: I don't see any other project having this file. Why do you have this file?

**Comment by vrmurthy01** [comment]
The file applicationhost.config was an autogenerated one by VS & seems unnecessary. I removed it now.

**Comment by azurecla** [comment]
Hi __@vrmurthy01__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (vrmurthy). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by stankovski** [comment]
@vrmurthy01 please pull from upstream and update the PR

**Comment by vrmurthy01** [comment]
Pulled from upstream & updated PR

**Pull request merged**